File: 240102/체질량지수 2/bmi-2.py
h, w = map(int, input().split())

# 키(cm)에서 키(m)로 단위 환산을 한 뒤 
# 체질량지수 계산 식에 넣어야 함에 유의합니다.
bmi = w * 100 * 100 // (h * h)

# 출력
print(bmi)
if bmi >= 25:
	print("Obesity")

File: 241009/체질량지수 2/bmi-2.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int cm = sc.nextInt();
        int kg = sc.nextInt();

        int bmi = kg*100*100/(cm*cm);

        if(bmi >= 25) {
            System.out.println(bmi);
            System.out.println("Obesity");
        }
        else {
            System.out.println(bmi);
        }
    }
}